pytest_inmanta/plugin.py

```diff
@@ -402,8 +404,14 @@ class Project:
     environment variable. Repositories are separated with spaces.
     """
 
-    def __init__(self, project_dir: str, load_plugins: bool = True) -> None:
+    def __init__(self, project_dir: str, env_path: str, load_plugins: typing.Optional[bool] = True) -> None:
+        """
+        :param project_dir: Directory containing the Inmanta project.
+        :param env_path: The path to the venv to be used by the compiler.
+        :param load_plugins: Load plugins iff this value is not None.
```

Comment:
I think this is a bug. As far as I can see both the `project_shared` and `project_shared_no_plugins` load plugins now. I don't think this is an issue, since this flag is planned to be removed using #116.